packages/core/test/Cardano/util/address.test.ts

```diff
@@ -1,11 +1,6 @@
-import * as parseCmlAddress from '../../../src/CML/parseCmlAddress';
```

Comment:
This test suite seems to be testing `Address.ts`, so would be best to consolidate the tests in 1 file